README.md

```diff
@@ -54,15 +52,18 @@ APIClarity supports integrating with the following taffic sources. Install APICl
    ```shell
    helm repo add apiclarity https://apiclarity.github.io/apiclarity
    ```
+2. Save APIClarity default chart values
```

Comment:
Save -> Backup?

Reply:
Maybe this is step is not needed and better edit the original (it's git after all :) )

Reply:
`helm show values` is a way to show the default chart values w/o cloning anything - it is not related to git

Reply:
Let's keep this step, it's a good practice to see & save the values before install.